storybook/stories/CampaignLocationScreen/index.tsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import { storiesOf } from "@storybook/react-native";
+import { View } from "react-native";
+import { CampaignLocationsScreen } from "../../../src/components/CampaignLocations/CampaignLocationsScreen";
+import { navigation, mockReactNavigationDecorator } from "../mocks/navigation";
+import { CampaignConfigsStoreContext } from "../../../src/context/campaignConfigsStore";
+import { AuthStoreContext } from "../../../src/context/authStore";
+import {
+  defaultFeatures,
+  defaultProducts,
+} from "../../../src/test/helpers/defaults";
+
+storiesOf("Screen", module)
+  .addDecorator((Story: any) => (
+    <AuthStoreContext.Provider
+      value={{
+        hasLoadedFromStore: true,
+        authCredentials: {
+          "Campaign 1": {
+            operatorToken: "operatorToken",
+            sessionToken: "sessionToken",
+            expiry: new Date().setDate(new Date().getDate() + 1),
```

Comment:
Can replace with this instead
```suggestion
            expiry: Date.now() + 86400000 // 1 day in millisecond,
```

Reply:
thank you. will update like that